app/views/shared/_header_navbar.html.erb

```diff
@@ -1,46 +0,0 @@
-<nav class="navbar navbar-expand-md navbar-dark bg-dark topbar" role="navigation">
-  <div class="<%= container_classes %>">
-    <%= link_to application_name, root_path, class: 'mb-0 navbar-logo' %>
-    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#user-util-collapse" aria-controls="user-util-collapse" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-
-    <div class="collapse navbar-collapse justify-content-md-end" id="user-util-collapse">
-      <%= render 'shared/user_util_links' %>
```

Comment:
Pretty sure we still have this with https://github.com/projectblacklight/blacklight/blob/deb5c53b20552b4cba148b0002844e14439acf59/app/components/blacklight/top_navbar_component.html.erb